_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -87,7 +87,7 @@ public static function register_endpoints() {
 		register_rest_route( 'jetpack/v4', '/connection/user', array(
 			'methods' => WP_REST_Server::EDITABLE,
 			'callback' => __CLASS__ . '::unlink_user',
-			'permission_callback' => __CLASS__ . '::link_user_permission_callback',
+			'permission_callback' => __CLASS__ . '::unlink_user_permission_callback',
```

Comment:
From  the pull summary,

> Removes `Jetpack_Core_Json_Api_Endpoints::link_user_permission_callback` which is never used.

Technically, this change stops using it in lieu of `Jetpack_Core_Json_Api_Endpoints::unlink_user_permission_callback` -- which had previously been unused, and then drops the formerly used one.